common/src/main/java/com/wynntils/models/containers/containers/CosmeticContainer.java

```diff
@@ -13,12 +13,21 @@
 import java.util.regex.Pattern;
 
 public class CosmeticContainer extends Container implements SearchableContainerProperty {
+    private static final String COSMETIC_PREVIEW_TITLE = "\uF029";
```

Comment:
```suggestion
    private static final String COSMETIC_PREVIEW_TITLE_CHARACTER = "\uF029";
```

Took me some time to figure out this was not the complete title.